src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -360,6 +351,37 @@ private void InitializeHelix()
             }
         }
 
+        void RegisterRenderDynamoEffect(EffectInitializationEventArgs args)
```

Comment:
This method is the event handler for a new event added to Helix which allows for third-party registration of `Effect` objects.